### PR TITLE
Add examples for SupervisedData class

### PR DESCRIPTION
**Pull request by yaz-saleh**
- Add examples to SupervisedData class
- Fix minor typos

### CODE REVIEW

File: pymleda/pymleda.py [hunk 1]
```diff
@@ -16,7 +16,9 @@ class SupervisedData:
         Sequence of target names (y) to be used as dependent variables or labels
 
     **kawrgs: 
-        Additional parameters to pass to sklearn's train_test_split(). For more information see hyperlink: `sklearn's function documentation <https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html/>`_.
+        Additional parameters to pass to sklearn's train_test_split(). 
+        In the absence of additional parameters for the default paramteres of train_test_split() are used including test size  = 0.25
```
Comment (Saule-Atymtayeva) [inline pymleda/pymleda.py:20]: Just a small typo. It should be "parameters" instead of "paramteres".

**Review by Saule-Atymtayeva — APPROVED** [review]

File: pymleda/pymleda.py (hunk 1 above)
Comment (yaz-saleh) [inline pymleda/pymleda.py:20]:
```suggestion
        In the absence of additional parameters for the default parameters of train_test_split() are used including test size  = 0.25
```

File: pymleda/pymleda.py [hunk 2]
```diff
@@ -16,7 +16,9 @@ class SupervisedData:
         Sequence of target names (y) to be used as dependent variables or labels
 
     **kawrgs: 
-        Additional parameters to pass to sklearn's train_test_split(). For more information see hyperlink: `sklearn's function documentation <https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html/>`_.
+        Additional parameters to pass to sklearn's train_test_split(). 
+        In the absence of additional parameters for the default parameters of train_test_split() are used including test size  = 0.25
```
Comment (yaz-saleh) [inline pymleda/pymleda.py:20]:
```suggestion
        In the absence of additional parameters, the default parameters of train_test_split() are used including test size  = 0.25
```

File: pymleda/pymleda.py (hunk 1 above)
Comment (yaz-saleh) [reply pymleda/pymleda.py]: Thank you - I always make that mistake! Fixed.

**Pull request merged**